.travis.yml

```diff
@@ -5,10 +5,26 @@ jdk:
     - openjdk8
     - oraclejdk8
     - oraclejdk9
-
+matrix:
+    exclude:
```

Comment:
Why are these excludes here (sorry if you already explained)?

Reply:
18.0.0.3 is less compatible with Java 9 than 18.0.0.2, due to changes in preparation for Java 11 support. As a result, we don't test 18.0.0.3 with Java 9 in LMP. I think we should do the same here.